test/unit/mock/profiles/exception-in-control/inspec.yml

```diff
@@ -0,0 +1,8 @@
+name: profile-01
+title: InSpec Profile
+maintainer: The Authors
```

Comment:
Some of the other `inspec.yml` (all others?) use different metadata values. Not sure if it important though.

See: https://github.com/chef/inspec/blob/master/test/unit/mock/profiles/aws-profile/inspec.yml

Reply:
I just used whatever `inspec init profile` generated.  Nothing in the metadata was relevant to the test underway.

Reply:
I figured, I'll mark it as approved, I don't think the copyright lines and such really matter. Was just going for consistency.